generators/client/templates/react/src/test/javascript/spec/app/shared/reducers/authentication.spec.ts.ejs

```diff
@@ -159,6 +157,32 @@ describe('Authentication reducer tests', () => {
         isAuthenticated: false
       });
     });
+  });
 
+  describe('Actions', () => {
+    let store;
+
+    const resolvedObject = { value: 'whatever' };
+    beforeEach(() => {
+      const mockStore = configureStore([thunk, promiseMiddleware()]);
+      store = mockStore({});
+      axios.get = sinon.stub().returns(Promise.resolve(resolvedObject));
+      axios.put = sinon.stub().returns(Promise.resolve(resolvedObject));
```

Comment:
unwanted stubs

Reply:
Indeed, I might have forgot this file. :sweat_smile: